README.md

```diff
@@ -64,13 +64,12 @@ After you're finished please remove all the comments and instructions!
 
 > Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[your_project__name]** is a... Microverse-Student-Project-1
```

Comment:
I can see you have done a great job creating your readme👍 but your readme file still lacks some important features
- [ ] Make sure you removed **template logo**. You can replace it with your own logo if you have one.
- [ ] Make sure you add the project name.
- [ ] Make sure you add the project description
- [ ] Make sure you add the information about how to  getting Started on your project
      - How to install
      - How to Setup
      - How to Usage